assets/src/onboarding-wizard/pages/done/index.js

```diff
@@ -0,0 +1,194 @@
+							{ __( 'In Transitional mode AMP and non-AMP versions of your site are served using your currently active theme.', 'amp' ) }
+						</p>
+						<p>
+							{ __( 'Browse your site below to ensure it meets your expectations, and toggle the AMP setting to compare both versions.', 'amp' ) }
+						</p>
+					</>
+				) }
+				{ READER === themeSupport && (
+					<>
+						<p>
+							{ __( 'In Reader mode AMP is served using your selected Reader theme, and pages for your non-AMP site are served using your primary theme. Browse your site below to ensure it meets your expectations, and toggle the AMP setting to compare both versions.', 'amp' ) }
+						</p>
+						<p>
+							{ __( 'As a last step, use the Customizer to tailor the Reader theme as needed.', 'amp' ) }
+						</p>
+					</>
+				) }
+				<div className="done__links-container">
+					<NavMenu
+						links={ previewLinks }
+						onClick={ ( e, link ) => {
+							e.preventDefault();
+							setActivePreviewLink( link );
+						} }
+					/>
+				</div>
+			</div>
+			<div className="done__preview-container">
+				{ READER === themeSupport && downloadingThemeError && (
+					<AMPNotice size={ NOTICE_SIZE_LARGE } type={ NOTICE_TYPE_INFO }>
+						{ __( 'There was an error downloading your Reader theme. As a result, your site is currently using the legacy reader theme. Please install your chosen theme manually.', 'amp' ) }
+					</AMPNotice>
+				) }
+				{ STANDARD !== themeSupport && (
+					<AMPSettingToggle
+						text={ __( 'AMP', 'amp' ) }
+						checked={ isPreviewingAMP }
+						onChange={ toggleIsPreviewingAMP }
+						compact={ true }
+					/>
+				) }
+				<Preview url={ previewUrl } />
+			</div>
+			<div className="done__content done__content--secondary">
+				<h2 className="done__icon-title">
+					<IconLaptopToggles />
+					{ __( 'Need help?', 'amp' ) }
+				</h2>
+				<ul className="done__list">
+					{ /* dangerouslySetInnerHTML reason: Injection of a link. */ }
+					<li dangerouslySetInnerHTML={ {
+						__html: sprintf(
+							/* translators: placeholder is a link to support forum. */
+							__( 'Reach out in the <a href="%s" target="_blank" rel="noreferrer noopener">support forums</a>', 'amp' ),
+							'https://wordpress.org/support/plugin/amp/#new-topic-0',
+						),
+					} } />
+					{ /* dangerouslySetInnerHTML reason: Injection of a link. */ }
+					<li dangerouslySetInnerHTML={ {
+						__html: sprintf(
+							/* translators: placeholder is a link to the settings page. */
+							__( 'Try a different template mode <a href="%s" target="_blank" rel="noreferrer noopener">in settings</a>', 'amp' ),
+							SETTINGS_LINK,
+						),
+					} } />
+					{ /* dangerouslySetInnerHTML reason: Injection of a link. */ }
+					<li dangerouslySetInnerHTML={ {
+						__html: sprintf(
+							/* translators: placeholder is a link to the plugin site. */
+							__( '<a href="%s" target="_blank" rel="noreferrer noopener">Learn more</a> how the PX plugin works', 'amp' ),
```

Comment:
Since we haven't gotten to that point yet:

```suggestion
							__( '<a href="%s" target="_blank" rel="noreferrer noopener">Learn more</a> how the AMP plugin works', 'amp' ),
```